.github/workflows/ci.yml

```diff
@@ -445,14 +433,12 @@ jobs:
         include:
           - name: vs2019_static
             compiler: cl
-            meson_options: --default-library=static -Dstatic_runtime=true
+            meson_options: --default-library=static -Db_vscrt=static_from_buildtype
```

Comment:
review if it's really necessary `static_runtime`. Maybe just remove in favour of this.